docs/docs/migrate-from-v1/README.md

```diff
@@ -0,0 +1,358 @@
+
+However, after deploying it's not possible to rollback to a prior version of a deployed workflow.
+
+You can still edit a deployed workflow, just like in v1 but automatic version rollbacks are not currently possible.
+
+## FAQs
+
+### What are the benefits of the new (v2) workflow builder?
+
+* **Edit & test** your workflows in separate editing mode without impacting live workflows
+* **Support for multiple languages** including Node, Python, Golang & bash
+* **Granular testing** including the ability to test individual steps and more
+* **Multiple triggers** are now supported per workflow
+* **Improved** forms for easier configuration and streamlined building
+
+### What are the limitations of the new (v2) workflow builder?
+
+* `$checkpoint` has been removed from v2 workflows, but `$.database` is a near equivalent.
+* Sharing workflows is not supported
+* Making workflows public is not supported
+* Workflows are no longer versioned
+
+[The limitations of the new workflow builder are covered in detail here](/docs/migrate-from-v1#limitations).
+
+### Are v2 workflows backwards compatible?
+
+No, v2 workflows are not currently compatible with the v1 builder.
+
+However, pre-built component actions are still compatible across both versions.
+
+### Is the Component API changing as well? Will I need to rewrite Components?
+
+No. Any components in the public registry or any private components you have published in your account are compatible with v2.
+
+The v2 workflow builder utilizes the same Component API allowing you to create components from within your workflows, which was not possible in v1.
+
+### Will I still be able to open and edit v1 workflows?
+
+Yes, absolutely you will still be able to view and edit v1 workflows. There is no need to immediately change your workflows from v1 to v2.
+
+### How do I migrate v1 workflows to v2 workflows?
+
+At this time we do not have an automated process to change v1 to v2. To create a v2 equivalent workflow, you can recompose your v1 workflow in the v2 builder.
+
+However, if it uses custom Node.js code steps, be sure to [follow the changes we describe in the guide above](/migrate-from-v1/#node-js-code-step-changes).
+
+### When will the new (v2) workflow builder be the default builder for all customers?
+
+v2 is currently an opt in feature, you can still create and edit v1 workflows if you have an grandfathered account.
+
+We will not make the v2 builder the default workflow experience until we have reached feature parity with v1 by providing paths to overcome the [limitations of v2](#known-gaps-limitations).
+
+### When will I no longer be able to create v1 workflows?
+
+There is currently no deprecation date for v1 workflows, we will continue to support of v1 workflows until we have feature parity with v2.
+
+When this date becomes clear we will provide assistance to automatically and assist migrate v1 to v2 workflows for you.
```

Comment:
Since we have no specific plan for automated migration yet, I would remove this section, and we can address that later.